assets/queries/terraform/gcp/stackdriver_logging_disabled/query.rego

```diff
@@ -4,30 +4,31 @@ import data.generic.terraform as tf_lib
 
 CxPolicy[result] {
 	resource := input.document[i].resource.google_container_cluster[primary]
-	not resource.logging_service
+	resource.logging_service == "none"
 
 	result := {
 		"documentId": input.document[i].id,
 		"resourceType": "google_container_cluster",
 		"resourceName": tf_lib.get_resource_name(resource, primary),
 		"searchKey": sprintf("google_container_cluster[%s]", [primary]),
-		"issueType": "MissingAttribute",
-		"keyExpectedValue": "Attribute 'logging_service' should be defined",
-		"keyActualValue": "Attribute 'logging_service' is undefined",
+		"issueType": "IncorrectValue",
+		"keyExpectedValue": "Attribute 'logging_service' should be undefined or 'logging.googleapis.com/kubernetes'",
+		"keyActualValue": "Attribute 'logging_service' is 'none'",
 	}
 }
 
+# legacy stackdriver was decomissioned Mar 31, 2021 https://cloud.google.com/stackdriver/docs/deprecations/legacy
 CxPolicy[result] {
 	resource := input.document[i].resource.google_container_cluster[primary]
-	resource.logging_service == "none"
+	resource.logging_service == "logging.googleapis.com"
 
 	result := {
 		"documentId": input.document[i].id,
 		"resourceType": "google_container_cluster",
 		"resourceName": tf_lib.get_resource_name(resource, primary),
 		"searchKey": sprintf("google_container_cluster[%s]", [primary]),
```

Comment:
```suggestion
		"searchKey": sprintf("google_container_cluster[%s].logging_service", [primary]),
```

Reply:
Rejected, this causes tests to fail, I think in the test case where `logging_service` is undefined?  When I reverted the change, anyways, the tests passed...(also the == "none" test was originally using the searchKey without the .logging_service portion)

Reply:
But to return the result in this rule `.logging_service` needs to be defined. If not defined there is no result for this rule.